src/main/logic/Logic.cpp

```diff
@@ -361,3 +375,19 @@ void Logic::gameEnd(){
 	mobileObjects.clear();
 	allObjects.clear();
 }
+
+void Logic::checkEnd(){
+	if(numLives <= 0){
+		bool end = TRUE;
```

Comment:
This should be `true` or `false`. `TRUE` is a typedef often used in C. I'm not sure which header it comes from, maybe `windows.h`, so it might not compile on *nix systems?